iothub/device/src/IotHubModuleClient.cs

```diff
@@ -87,8 +101,13 @@ internal IotHubModuleClient(IotHubConnectionCredentials iotHubConnectionCredenti
         /// Creates a disposable <c>IotHubModuleClient</c> instance in an IoT Edge deployment based on environment variables.
         /// </summary>
         /// <param name="options">The options that allow configuration of the module client instance during initialization.</param>
-        /// <exception cref="InvalidOperationException">The required environmental variables were missing. Check the exception thrown for additional details.</exception>
         /// <returns>A disposable client instance.</returns>
+        /// <exception cref="InvalidOperationException">The required environmental variables were missing. Check the exception thrown for additional details.</exception>
+        /// <example>
+        /// <code language="csharp">
+        /// await using var client = await IotHubModuleClient.CreateFromEnvironmentAsync(new IotHubClientOptions(new IotHubClientMqttSettings(IotHubClientTransportProtocol.WebSocket)));
+        /// </code>
+        /// </example>
         public static async Task<IotHubModuleClient> CreateFromEnvironmentAsync(IotHubClientOptions options = default)
```

Comment:
Any reason why we don't have a cancellation token in this async method?

Not for your PR, Bryce, but maybe we should add a task to add one here.